tastyworks/models/watchlists.py

```diff
@@ -0,0 +1,95 @@
+import aiohttp
+from tastyworks.models.session import TastyAPISession
+
+
+class Watchlist(object):
+    def __init__(self, name=None):
+        self.name = name
+        self.group_name = None
+        self.securities = {}
+
+    @classmethod
+    def from_list(cls, data: list):
+        """From List
+
+        Generator for making a watchlist.
+
+        Notes:
+            There is an instability issue with Tastyworks API.  Hence
+            the need to catch an exception.  Special symbols don't have
+            instrument-type's at all.
+
+        Args:
+            data (list):   List of dict objects containing symbol &
+                           [instrument-type | instrument_type] keys.
+
+        Returns:
+            Watchlist:  A watchlist object
+        """
+        inst = cls()
+        for item in data:
+            try:
+                inst.securities[item['symbol']] = {
+                    'instrument-type': item['instrument-type']
+                }
+            except KeyError:
+                try:
+                    inst.securities[item['symbol']] = {
+                        'instrument_type': item['instrument_type']
+                    }
+                except KeyError:
+                    inst.securities[item['symbol']] = {
+                        'instrument-type': None
+                    }
+        return inst
+
+
+class WatchlistGroup(object):
+    """Watchlist Group
+
+    Generator for making document of watchlist objects.
+
+    Notes:
+        Not all watchlists return group-name key's.  Hence the need to catch
+        an exception.
+
+    Args:
+        session (TastyAPISession): Tastyworks API Session object
+        public (boolean): True if retrieving public-watchlists, false if
+                          retrieving custom-watchlists
+
+    Returns:
+        WatchlistGroup: A group of watchlists
+
+    """
+    def __init__(self):
+        self.watchlists = {}
+
+    async def load_watchlists(
+        self, session: TastyAPISession, public: bool = True
+    ):
+        url = f'{session.API_url}/public-watchlists' if public else f'{session.API_url}/watchlists'  # NOQA: E501
```

Comment:
What does the `# NOQA: E501` signify?

Reply:
E501 is for "line too long".  For an API URL, I figured it would be better served with a lint exception rather than obfuscating the URL by breaking it up.

In its current state, this is more of a catch-all since there are technically two watchlist endpoints - one public and one private.  I'm willing to break it up if given a good use case.

Reply:
Ah right, a linter directive. Looks perfectly readable to me 👍

Reply:
I guess I should ask first - is flake8 the linter of choice for now?

Reply:
I used flake8 for linting when writing this API. I have no strong opinions and happy to change it to another (popular) linter if you want

Reply:
I'm a fool.  Last night I was tinkering around and noticed the `tox.ini` file omits `E501` by default.  This made me realize there was actually an issue with how I was linting to begin with, I went ahead and removed all my references.